node/src/index.ts

```diff
@@ -94,6 +94,12 @@ if (process.env.EXTISM_PATH) {
 
 const lib = locate(searchPath);
 
+// get the true byte size of a utf-8 string
+const byteSize = (data: string | Buffer) => {
```

Comment:
perhaps instead of doing this we can just always turn the string into a buffer. There might be a better way to do this.

Reply:
It looks like there is also `Buffer.byteLength`: https://nodejs.org/api/buffer.html#static-method-bufferbytelengthstring-encoding

Reply:
Thanks i'll try that

Reply:
Seems to work!